clusterloader2/testing/load/config.yaml

```diff
@@ -132,18 +132,10 @@ steps:
       action: start
       checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
       replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
-  - Identifier: DnsLookupLatency
-    Method: DnsLookupLatency
+  - Identifier: SLOMeasurement
+    Method: SLOMeasurement
     Params:
-      action: start
-      checkProbesReadyTimeout: {{$PROBE_MEASUREMENTS_CHECK_PROBES_READY_TIMEOUT}}
-      replicasPerProbe: {{AddInt 2 (DivideInt .Nodes 100)}}
-  {{if $PROMETHEUS_SCRAPE_KUBE_PROXY}}
-  - Identifier: NetworkProgrammingLatency
-    Method: NetworkProgrammingLatency
-    Params:
-      action: start
-  {{end}}
+      action:start
```

Comment:
The other params (checkProbesReadyTimeout etc. are still needed - they are use by the measurement so we need to pass them.